app/backend/templates/email_changed_notification.md

```diff
@@ -1,5 +1,5 @@
 ---
-subject: "Your Couchers.org email was changed"
+subject: "Couchers.org email change requested"
 ---
 
 Hi {{ escape(user.name) }}!
```

Comment:
>Your email on Couchers.org was changed.

This email is ambiguous about the process. Was it requested, or did it happen?

Additionally, do we confirm with the original address before confirming with the new one? Like if I'm a hacker and I know someone's coucher login info, can I change their email, confirm via my email address, and then use their account, while the account owner has to contact support?

Reply:
> Was it requested, or did it happen?

Fixing

> Additionally, do we confirm with the original address before confirming with the new one?

No

> Like if I'm a hacker and I know someone's coucher login info, can I change their email, confirm via my email address, and then use their account, while the account owner has to contact support?

Yes

Reply:
Yeahhh, this is a problem when people don't have a password, need to add that.

Reply:
Opened #912, that can be sorted out there.

Reply:
>Yes

And we're ok with this?

Reply:
For now we are, but I've opened a high priority issue in #912.